WebApp/autoreduce_webapp/templates/submit_runs.html

```diff
@@ -73,19 +73,15 @@ <h2>{{ instrument.name }} - Re-run past jobs</h2>
                                     <li>
                                         <a href="#currentScript" id="currentScript">Reset to values in current script</a>
                                         <div class="js-explaination visible-xs-block">Reset all variables to those set in the script. This will pick up any changes made to the reduce_vars.py script.</div>
-                                    </li>
-									<li>
-                                        <input type="checkbox" name="overwrite_checkbox" id="overwrite_checkbox" checked>
-                                        <a href="#overwrite" id="overwrite"> Overwrite Previous Data</a>
-                                        <div class="js-explaination visible-xs-block">Uncheck this box if you don't want previous run data to be overwritten</div>
                                     </li>
                                     <li>
                                         <input type="checkbox" name="overwrite_checkbox" id="overwrite_checkbox" checked>
-                                        <a href="#overwrite" id="overwrite"> Overwrite Previous Data</a>
-                                        <div class="js-explaination visible-xs-block">
-											By default, when an autoreduction job is re-run, the original data from the initial run will be overwritten. If you uncheck this box,
-											original data will not be overwritten and you will have distinct folders to separate out the data output from your different re-runs.
-										</div>
+                                        <label for="overwrite_checkbox">
+                                            Overwrite Previous Data
+                                            <a href="#" data-toggle="popover" data-content="By default, when an autoreduction job is re-run, the original data from the initial run will be overwritten. If you uncheck this box,
+											   original data will not be overwritten and you will have distinct folders to separate out the data output from your different re-runs."
```

Comment:
Very minor but could the indenting for this line of text be lined up to the rest of the text?

Reply:
Strange I thought it was. Maybe it went funny as it got pushed. I'll sort it

Reply:
Hi @keiranjprice101 reason is included tabs instead of spaces. Replace tabs with spaces. Also, in your IDE editor recommend setting tabs to be automatically replaced by spaces